build.gradle.kts

```diff
@@ -1,41 +1,10 @@
-buildscript {
-    val kotlinVersion by extra("1.3.61")
-    extra.set("gdxVersion", "1.9.10")
-    extra.set("roboVMVersion", "2.3.7")
-    extra.set("box2DLightsVersion", "1.4")
-    extra.set("ashleyVersion", "1.7.3")
-    extra.set("aiVersion", "1.8.2")
-    extra.set("ktxVersion", "1.9.10-b2")
-    val agpVersion by extra("3.5.3")
-
-    repositories {
-        mavenLocal()
-        mavenCentral()
-        maven("https://plugins.gradle.org/m2/")
-        maven("https://oss.sonatype.org/content/repositories/snapshots/")
-        jcenter()
-        google()
-    }
-
-    dependencies {
-        classpath("com.android.tools.build:gradle:$agpVersion")
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-    }
+plugins {
+    kotlin("jvm") version Versions.kotlin apply false // root project does not need Kotlin plugin since it does not contain any source
```

Comment:
Is this needed in root project as it is anyway specified in `core` and `desktop` project?

Reply:
It's more clean to have the version only at a single place 
The sub projects only use `kotlin("jvm")` and inherit the version from it's parent